modules/distribution/src/main/scripts/impl/init.sh

```diff
@@ -27,4 +27,13 @@ script=$( basename "$SOURCE" )
 
 . "$impl"/config.sh
 
+if [[ -z $HADOOP_PREFIX ]]; then
+  echo "HADOOP_PREFIX needs to be set!"
+  exit 1
+fi
+
+$HADOOP_PREFIX/bin/hdfs dfs -mkdir -p /fluo/lib
+$HADOOP_PREFIX/bin/hdfs dfs -copyFromLocal $FLUO_HOME/lib/fluo-api-*.jar /fluo/lib/
```

Comment:
Talked about this offline.  The properties file specifies each jar so multiple jars is not an issue.  Clearing the lib/ directory in HDFS could affect other Fluo instances (if multiple are running).  It was decided to leave this code here but add an echo telling the user that files are being copied to HDFS.